tsconfig.json

```diff
@@ -9,7 +9,7 @@
 			"*": ["typings/*"]
 		}
 	},
-	"include": ["src/**/*.ts"],
+	"include": ["src/**/*.ts", "src/components/Slider/Slider.tsx", "src/components/Slider/index.tsx"],
```

Comment:
why was this done?

Reply:
Sorry, was just checking something with the file extensions. Had no idea tsconfig.json added these lines by itself. Fixed it and pushed

Reply:
always review each file before committing.

Reply:
Ok sir.